src/Form/LegacyConsumer/templates/select.html.php

```diff
@@ -1,20 +1,26 @@
 <?php /** @var Give\Framework\FieldsAPI\Select $field */ ?>
 <select
-	name="give_<?php echo $field->getName(); ?>"
+	name="<?php echo $field->getName(); ?><?php echo $field->getAllowMultiple() ? '[]' : ''; ?>"
 	id="give-<?php echo $field->getName(); ?>"
 	<?php echo $field->getAllowMultiple() ? 'multiple' : ''; ?>
 	<?php echo $field->isRequired() ? 'required' : ''; ?>
 	<?php echo $field->isReadOnly() ? 'readonly' : ''; ?>
 >
-	<?php if ( $placeholder = $field->getPlaceholder() ) : ?>
-		<option value=""><?php echo $placeholder; ?></option>
-	<?php endif; ?>
+	<?php
+	if ( $placeholder = $field->getPlaceholder() ) {
+		printf(
+			'<option value="" disabled %2$s>%1$s</option>',
+			$placeholder,
+			$field->getDefaultValue() ? '' : 'selected'
+		);
+	}
+	?>
 	<?php foreach ( $field->getOptions() as $option ) : ?>
-		<?php $label = $option->getLabel(); ?>
 		<?php $value = $option->getValue(); ?>
+		<?php $label = $option->getLabel(); ?>
 		<?php $default = $field->getDefaultValue() === $option->getValue(); ?>
 		<option
-			<?php echo $label ? "value={$value}" : ''; ?>
+			<?php echo "value=\"{$value}\""; ?>
```

Comment:
I think we could go either way here, right? For `<option>` the `value` attribute can be omitted and it will just use the `textContent` of the element as the `value`.

Reply:
@knowler i reverted change: https://github.com/impress-org/givewp/commit/62ee48bf965ca33f114c8bcec701fa6b0b33b5ad